modules/ossm-config-disable-networkpolicy.adoc

```diff
@@ -15,11 +13,16 @@ If you want to disable the automatic creation and management of `NetworkPolicies
 When you disable `spec.security.manageNetworkPolicy` {ProductName} will not create *any* `NetworkPolicy` objects.  The system administrator is responsible for managing the network and fixing any issues this might cause.
 ====
 
+.Prerequisites
+
+* {ProductName} Operator version 2.1.1 or higher installed.
+* `ServiceMeshControlPlane` resource updated to version 2.1.1 or higher.
```

Comment:
SMCP's version is in the form `v2.0`, `v2.1`, etc. So this statement could be reworded to something like:

`ServiceMeshControlPlane` resource updated to version `v2.1`.